megatron/tools/run_text_generation_uvicorn_server_single_thread.py

```diff
@@ -179,7 +173,7 @@ def postprocessing(fun):
                 lock_stream.release()
 
             return StreamingResponse(trans(), media_type="text/plain",
-                                background=BackgroundTask(postprocessing, fun), lock=lock_stream)
+                                background=BackgroundTask(postprocessing, fun))
```

Comment:
Same question as above.

Reply:
This issue was left over from before, and originally there was no lock=lock_stream added. Whether there is lock=lock_stream or not, the effect is the same.